OWNERS

```diff
@@ -0,0 +1,20 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- alexeykazakov
+- MatousJobanek
+- sbryzak
+- xcoulon
+- rajivnathan
+- ranakan19
+- mfrancisc
+
+reviewers:
+- alexeykazakov
+- MatousJobanek
+- michaelkleinhenz
```

Comment:
I'm not sure if @michaelkleinhenz still wants review our PRs? ;)

Reply:
he is added as CODEOWNER, so was trying to make OWNER  file same as it

Reply:
a miss,  he is not there as codeowner.. so should we remove his name or keep his name here ?